### PR TITLE
Add an example for IO.gets

### PR DESCRIPTION
**Pull request by radar**

### CODE REVIEW

File: lib/elixir/lib/io.ex [hunk 1]
```diff
@@ -259,6 +259,12 @@ defmodule IO do
     * `{:error, reason}` - other (rare) error condition;
       for instance, `{:error, :estale}` if reading from an
       NFS volume
+
+  Example:
```
Comment (josevalim) [inline lib/elixir/lib/io.ex]: ## Examples

File: lib/elixir/lib/io.ex [hunk 2]
```diff
@@ -259,6 +259,12 @@ defmodule IO do
...
+
+  To display "What is your name?" as a prompt and await user input:
+
+    IO.gets "What is your name?"
```
Comment (josevalim) [inline lib/elixir/lib/io.ex]: Four spaces please so it is shown as code.

**Comment by josevalim** [comment]
I have added two comments, thank you!

**Comment by radar** [comment]
I've addressed both of these comments. Thanks!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**